cloudant-client/src/main/java/com/cloudant/client/api/model/DesignDocument.java

```diff
@@ -446,6 +457,20 @@ public String getDbCopy() {
             return dbcopy;
         }
 
+        /**
+         * @param element JSON object or string
+         * @return a string form of the object or the JSON string with leading and trailing "
+         * stripped
+         */
+        private String stringifyMRElement(JsonElement element) {
+            String s = element.toString();
```

Comment:
Wouldn't this throw an exception if the element is `null`? Also, what about renaming the variable `s` to show that the variable holds the string representation of the JSON element?